lightpath/tests/conftest.py

```diff
@@ -25,7 +26,7 @@
 def pytest_addoption(parser):
     parser.addoption("--log", action="store", default="INFO",
                      help="Set the level of the log")
-    parser.addoption("--logfile", action="store", default=None,
+    parser.addoption("--logfile", action="store", default='log',
```

Comment:
Are you finding these command line arguments useful?

Reply:
No no at all. Should be refactored to the `/logs` folder setup. That commit got snuck in.

Reply:
Ah, okay. I was just fishing for more workflow improvement potential for other modules.